lib/texts.js

```diff
@@ -109,4 +109,8 @@ ${leaves}`,
 	'tl.finishedWithErrors': () => 'Some executions finished with errors',
 	'tl.finishedWithSuccess': () => 'All executions finished successfully',
 	'tl.promptPassword': () => 'Please enter your passwod: ',
+
+	// error messages
+	'err.msg.appRunning': 'App is still running',
+	'err.msg.conditionNotSatisfied': count => `Maximum amount of key presses ${count} reached. Condition was not satisfied`,
```

Comment:
we usually do it as `template`

Reply:
I think @JeKachan used this syntax already. It's OK with me as well as template function.